src/admin/index.js

```diff
@@ -69,6 +69,16 @@ module.exports = ({
     instrumentationEmitter.emit(DISCONNECT)
   }
 
+  /**
+   * @return {Promise}
+   */
+  const listTopics = async () => {
+    await cluster.refreshMetadata()
+    const { topicMetadata } = await cluster.metadata({ topics: [] })
```

Comment:
`{ topics: []}` is already the default behavior for `cluster.metadata`, so no need to pass it in.